charts/collab/values.yaml

```diff
@@ -206,13 +206,13 @@ serviceAccount:
 
 podAnnotations: {}
 
-securityContext: {}
+securityContext:
   # capabilities:
   #   drop:
   #   - ALL
-  # readOnlyRootFilesystem: true
   # runAsNonRoot: true
-  # runAsUser: 1000
+  readOnlyRootFilesystem: true
+  runAsUser: 0
```

Comment:
same here, may be there should be a comment to change to 0 if readonly file system?